libr/util/thread.c

```diff
@@ -44,7 +44,7 @@ R_API int r_th_push_task(struct r_th_t *th, void *user) {
 }
 
 R_API R_TH_TID r_th_self(void) {
-#if HAVE_PTRACE || __APPLE__
+#if HAVE_PTHREAD
```

Comment:
haha good one

Reply:
What was the reason for `HAVE_PTRACE || __APPLE__`? I don't see a reason why it shouldn't be  `HAVE_PTHREAD`.

Reply:
But doesn't matter anyway, I don't need r_th_self() anymore.